apis/infix/atrium-api-infix/src/jvmTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/BigDecimalExpectationSamples.kt

```diff
@@ -28,4 +29,44 @@ class BigDecimalExpectationSamples {
             expect(null as BigDecimal?) notToEqual null
         }
     }
+
+    @Test
+    fun toEqualNumerically(){
```

Comment:
please add the link to this and the other function in the KDOC (via `@sample`) of the corresponding functions in
apis/infix/atrium-api-infix/src/jvmMain/kotlin/ch/tutteli/atrium/api/infix/en_GB/bigDecimalExpectations.kt 

See toEqual